edx_solutions_api_integration/users/serializers.py

```diff
@@ -87,8 +87,8 @@ def get_profile_image(self, user):
 
     def get_courses_enrolled(self, user):
         """ Serialize user enrolled courses """
-        enrollments = user.courseenrollment_set.all()
-        return [unicode(enrollment.course_id) for enrollment in enrollments]
+        enrollments = user.courseenrollment_set.filter(is_active=1).values_list('course_id', flat=True)
```

Comment:
nit: replace 1 with `True` and rename `enrollments`, since the queryset is returning list of `course_id`